### PR TITLE
Integrate ditto-reader/opendss into workflow

### PR DESCRIPTION
**Pull request by kflemin**
### Pull Request Description
Adds the ability to run opendss workflow with the CLI.  Checks that a suitable version of python is installed and that the urbanopt-ditto-reader package is installed.  Runs the python workflow directly from the CLI via the pycall gem.

### CODE REVIEW

**Comment by vtnate** [comment]
I got useful error messages before installing the ditto reader regarding python and pip checks. After cloning and installing the ditto reader (inside a new venv) I got an elegant error saying the ditto reader wasn't found. After removing the venv and reinstalling the ditto reader, I got the same error saying the ditto reader wasn't found.

**Comment by kflemin** [comment]
@vtnate, I haven't gotten it to work in a venv yet, but it works for me in a python version installed by pyenv (just don't create a virtual env).  if you pip list you should see the dittoreader installed.   you can also try setting a PYTHON env var to the path of your python to see if that will work. let me know!

**Comment by vtnate** [comment]
Jenkins failure is expected - this PR uses PyCall which isn't available to Jenkins.

File: uo_cli.gemspec [hunk 1]
```diff
@@ -33,6 +33,8 @@ Gem::Specification.new do |spec|
 
   #   use specific versions of urbanopt and openstudio dependencies while under heavy development
   spec.add_dependency 'optimist'
+  spec.add_dependency 'os', '~> 1.1.1'
+  spec.add_dependency 'pycall', '1.2.1'
```
Comment (vtnate) [inline uo_cli.gemspec:37]: I don't see `os` being used anywhere. Can we remove it?
Any reason we can't use pycall 1.3.1 instead?

**Comment by vtnate** [comment]
Commit a8a2049 addresses #123

**Comment by vtnate** [comment]
Test needs to be improved. Jenkins reads `Checking system.....`, not the output we're expecting in the current test. Test passes locally, but doesn't appear to be providing useful information. @kflemin can you help me write the test so it gets the correct bit of output that we want to read?

**Pull request merged**